components/arm/yahboom/dofbot.go

```diff
@@ -294,14 +295,14 @@ func (a *Dofbot) readJointInLock(ctx context.Context, joint int) (float64, error
 
 	time.Sleep(3 * time.Millisecond)
 
-	res, err := a.handle.ReadWordData(ctx, reg)
+	bytes, err := a.handle.ReadBlockData(ctx, reg, 2)
```

Comment:
[nit] use `rd` since `bytes` is a likely package name to be in use here at some point.

Reply:
excellent suggestion, thanks!